Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -230,6 +230,14 @@ PhysicalParticleContainer::PhysicalParticleContainer (AmrCore* amr_core, int isp
     }
 #endif
 
+    // If old particle positions should be saved add the needed components
+    pp_species_name.query("save_old_position", m_save_old_position);
+    if (m_save_old_position) {
+        AddRealComp("xold");
```

Comment:
I think this PR should be covered by a CI test that also adds CI, just to make sure the functionality will continue to work over time.
Can we add a PICMI or `inputs` based test to the `Examples/` and output via plotfiles and openPMD files?

Reply:
name: `prev_x`, `prev_y` would be picked up by openPMD as "vector" :)

Reply:
I have added the CI test (_Examples/Tests/ParticleDataPython/PICMI_inputs_prev_pos_2d.py_) and changed the names of the PIDs as well as the flag to turn on the new functionality.